### PR TITLE
Add zoom_control option to `Map()`

### PR DESCRIPTION
**Pull request by okomarov**
Enhancement #795: add a `zoom_control` option to the map creation (default: `True`)

Use case: create a .gif from 30+ snapshots of the map. The zoom controls do not serve any purpose and clutter the view. 

If averse to adding kwargs to the `Map()`  constructor, I could create a set method for ex-post toggle of the zoom controls.

### CODE REVIEW

File: folium/folium.py [hunk 1]
```diff
@@ -197,7 +199,8 @@ class Map(MacroElement):
         maxBounds: bounds,
         layers: [],
         worldCopyJump: {{this.world_copy_jump.__str__().lower()}},
-        crs: L.CRS.{{this.crs}}
+        crs: L.CRS.{{this.crs}},
+        zoomControl: {{this.zoom_control.__str__().lower()}}
```
Comment (Conengmo) [inline folium/folium.py]: please add a trailing comma after the line you added

**Comment by okomarov** [comment]
@Conengmo Thanks for the quick feedback. 

Travis is not failign a test per se but failing to import:
```
==================================== ERRORS ====================================
________ ERROR collecting tests/plugins/test_time_slider_choropleth.py _________
ImportError while importing test module '/tmp/tests/plugins/test_time_slider_choropleth.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/home/travis/miniconda/envs/TEST/lib/python2.7/site-packages/_pytest/python.py:468: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
/home/travis/miniconda/envs/TEST/lib/python2.7/site-packages/py/_path/local.py:668: in pyimport
    __import__(modname)
/home/travis/miniconda/envs/TEST/lib/python2.7/site-packages/_pytest/assertion/rewrite.py:216: in load_module
    py.builtin.exec_(co, mod.__dict__)
tests/plugins/test_time_slider_choropleth.py:16: in <module>
    import geopandas as gpd
/home/travis/miniconda/envs/TEST/lib/python2.7/site-packages/geopandas/__init__.py:4: in <module>
    from geopandas.io.file import read_file
/home/travis/miniconda/envs/TEST/lib/python2.7/site-packages/geopandas/io/file.py:3: in <module>
    import fiona
/home/travis/miniconda/envs/TEST/lib/python2.7/site-packages/fiona/__init__.py:69: in <module>
    from fiona.collection import Collection, BytesCollection, vsi_path
/home/travis/miniconda/envs/TEST/lib/python2.7/site-packages/fiona/collection.py:9: in <module>
    from fiona.ogrext import Iterator, ItemsIterator, KeysIterator
E   ImportError: libhdf5_cpp.so.102: cannot open shared object file: No such file or directory
```

This bug report suggests to uninstall and reinstall `h5py`: https://github.com/h5py/h5py/issues/672 

Ideas?

File: requirements-dev.txt [hunk 2]
```diff
@@ -10,6 +10,7 @@ flake8-quotes
 geographiclib
 geopandas
 gpxpy
+h5py
```
Comment (ocefpaf) [inline requirements-dev.txt]: I'm not sure this will fix things. I'm looking into the issue on the coda-forge side. Hold on...

Comment (ocefpaf) [reply requirements-dev.txt]: BTW, folium does not depend on `h5py` at all. The error is coming from `fiona` b/c of it links to `hdf5`. Please remove that line and wait a little bit as I fix the package upstream.

Comment (okomarov) [reply requirements-dev.txt]: Will revert commit.

**Comment by ocefpaf** [comment]
Fixing the package stack will take longer than I thought.
In #900 I skip the tests that uses `geopandas` in the `default` config and we can let the 3 notebooks that also use `geopandas` to just fail for now.

@okomarov you'll beed to rebase after we merge #900

**Comment by okomarov** [comment]
@ocefpaf Thanks for looking into this. Merged upstream master into current PR.

**Comment by okomarov** [comment]
So now everything passes except the notebooks that import geopandas.

**Comment by ocefpaf** [comment]
LGTM. Thanks @Conengmo for the review and @okomarov for the PR!

**Pull request merged**